docs/usage.rst

```diff
@@ -82,6 +83,9 @@ First, create your configuration file (e.g. ``/foo/bar/my_config.yml``):
         server_url: 'http://localhost:5341'
         api_key: 'your_api_key_if_you_have_one'
 
+        # Use a custom JSON encoder, if you need to.
+        json_encoder_class: json.encoder.JSONEncoder
```

Comment:
I think this might do it:

```python
def get_logger_class_by_name(name):
  return reduce(getattr, name.split("."), sys.modules[__name__])
```

Reply:
Oops - need some sleep; that's trying to be too clever, but yeah I can see how to resolve the class name. Will add functionality for it.

Reply:
Something like this:

```python
def get_class_by_name(name):
    name_parts = name.split('.')
    module_name = '.'.join(
        name_parts[:-1]
    )
    target_module = __import__(module_name)

    class_name = name_parts[-1:]
    target_class = getattr(target_module, class_name)

    return target_class
```

Will have a proper look at it tomorrow though once I've had some sleep

Reply:
How does https://github.com/tintoy/seqlog/pull/14/commits/167eb726d835dcf347810122a4beced40c5d9fff look?

Reply:
I have use several code to fix this bug:

```python3
def configure_from_file(file_name, override_root_logger=True, use_structured_logger=True):
    """your comment"""
    with open(file_name) as config_file:
        config = yaml.load(config_file)
    for name, handler in config.get('handlers', {}).items():
        if 'json_encoder_class' in handler:
            json_encoder_config = logging.config.BaseConfigurator({})
            config['handlers'][name]['json_encoder_class'] = json_encoder_config.resolve(handler['json_encoder_class'])

    configure_from_dict(config, override_root_logger)
```
I use the BaseConfigurator define in `logging.config` to replace the function get_class_by_name, because I think use the build_in module could avoid some unexpected bug

Reply:
Interesting, I'll give that a try, thanks!